webpack/ForemanTasks/Components/TasksDashboard/__tests__/__snapshots__/TasksDashboardActions.test.js.snap

```diff
@@ -32,6 +32,22 @@ Array [
 ]
 `;
 
+exports[`TasksDashboard - Actions should fetch-tasks-summary for subtasks and success 1`] = `
+Array [
+  Array [
+    Object {
+      "type": "FOREMAN_TASKS_DASHBOARD_FETCH_TASKS_SUMMARY_REQUEST",
+    },
+  ],
+  Array [
+    Object {
+      "payload": "some-data-sub-task",
```

Comment:
Could we use a real example of the given payload, e.g:
![Selection_235](https://user-images.githubusercontent.com/26363699/69040970-68520800-09f7-11ea-93d3-73d22b89c154.png)

Reply:
Added thanks